src/crypto/public_key/elliptic/ecdh.js

```diff
@@ -259,8 +269,9 @@ async function webPrivateEphemeralKey(curve, V, Q, d) {
  * Generate ECDHE ephemeral key and secret from public key using webCrypto
  *
  * @param  {Curve}                  curve        Elliptic curve object
- * @param  {Uint8Array}             Q                   Recipient public key
- * @returns {Promise<{V: Uint8Array, S: BN}>}   Returns public part of ephemeral key and generated ephemeral secret
+ * @param  {Uint8Array}             Q            Recipient public key
+ *
+ * @returns {Promise<{publicKey Uint8Array, sharedKey Uint8Array}>}
```

Comment:
```suggestion
 * @returns {Promise<{publicKey: Uint8Array, sharedKey: Uint8Array}>}
```
Some other functions too